README.md

```diff
@@ -71,6 +71,58 @@ It is required to set:
 `include` and `exclude` rules behave similarly to Webpack's loader options.  In addition to a RegExp you can pass a function which will be called with the path as its first argument.  Returning a truthy value will match the rule.  You can also pass an Array of rules, all of which must pass for the file to be included or excluded.
 
 
+##### Pathing for resources outside of output.path
+Resources that are located outside of the webpack output.path directory are pathed as follows
```

Comment:
I would change this to be super super clear:
`Resources that are compiled during webpack build outside of the webpack output.path directory are pathed as follows`.